operatorconfig/moduleconfig/authorization/v1.4.0/ingress.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: proxy-server
+  namespace: <NAMESPACE>
+spec:
+  ingressClassName: <PROXY_INGRESS_CLASSNAME>
```

Comment:
To support OpenShift, the ingressClassName shouldn't be specified so that the default ingress controller registers it. I don't think we are supporting OpenShift in this PR but just noting it.